ethereum.md

```diff
@@ -95,7 +95,7 @@ To do so, we'll extend sort `JSON` with some IELE specific syntax, and provide a
     rule <k> startTx => #finalizeBlock ... </k>
          <txPending> .List </txPending>
 
-    rule <k> startTx => loadTx(#sender(TN, TP, TG, TT, TV, #unparseByteStack(DATA), TW, TR, TS)) ... </k>
```

Comment:
Do we need to keep `#sender` around in the semantics anymore?

Reply:
I guess not, but there's some other code that's not currently being used laying around, it might be better to do it in a separate PR.

Reply:
OK cool.